main/res/values-el/strings.xml

```diff
@@ -1869,7 +1869,7 @@
     <string name="receivemapfile_cancelled">Η μεταφορά του αρχείου χάρτη ακυρώθηκε</string>
     <!-- Download map preference and menu entry -->
     <string name="downloadmap_title">Λήψη χαρτών εκτός σύνδεσης</string>
-    <string name="downloadmap_info">Επιλέξτε ένα αρχείο χάρτη για την περιοχή σας πατώντας το εικονίδιο \"Αποθήκευση\".\n\n 
+    <string name="downloadmap_info">Επιλέξτε ένα αρχείο χάρτη για την περιοχή σας πατώντας το εικονίδιο \"Αποθήκευση\".\n\n
```

Comment:
This looks unrelated.

Reply:
but correct, so lets merge

Reply:
Maybe my android Studio inserted a whitespace somewhere? Can't really see what happened here